### PR TITLE
Auto-lift sequence & Auto fixes

### PR DESCRIPTION
**Pull request by vladaviedov**
Sequence and commands for performing automatic lifting.
The starting position for the auto-lift is with the robot rolled up to the bar with hooks extended above it.
This branch also has some bug fixes related to autonomous code.

### CODE REVIEW

File: src/main/java/frc/robot/commands/FlywheelAutoRun.java [hunk 1]
```diff
@@ -51,6 +51,7 @@ public void execute() {
   /** Make this return true when this Command no longer needs to run execute()*/
   @Override
   public boolean isFinished() {
+    System.out.println(flywheelShootDurationTimer.get());
```
Comment (JurassikLizard) [inline src/main/java/frc/robot/commands/FlywheelAutoRun.java:54]: This never returns whether finished or not?

**Comment by JurassikLizard** [comment]
This seems very hacky, with a bunch of commented out stuff, but I suppose it's okay.

**Review by JurassikLizard — APPROVED** [review]

**Comment by vladaviedov** [comment]
@JurassikLizard This PR is not yet finished, much of the debug code will be removed later.

It is hacky, because we had a serious issue and we needed a very quick fix: basically the manual commands were running in autonomous mode and preventing auto commands from running properly. This was because setDefaultCommand doesn't care which mode the robot is running in.

As a temporary solution we load the commands when teleop is activated for the first time.

**Comment by JurassikLizard** [comment]
That's what I thought, and why I approved.

**Review by beryliumOfRecordRobotics — APPROVED** [review]

**Review by beryliumOfRecordRobotics — APPROVED** [review]

**Pull request merged**